extensions/resource-deployment/src/services/tools/toolBase.ts

```diff
@@ -27,17 +27,14 @@ const toolStatusLocalized: Map<ToolStatus, string> = new Map<ToolStatus, string>
 ]);
 
 export const enum dependencyType {
-	PythonAndPip3 = 'PythonAndPip3',
 	Brew = 'Brew',
```

Comment:
Should the strings be lowercase?

Reply:
It does not matter. They are not seen externally anywhere, but if you prefer, I can change them to lowercase. It was kept the same as enum value so it is easy to match the enum value.

Reply:
Nah no big deal, was just curious if we use them like that